Classes/GraphQL/Resolver/Type/AssetVariantResolver.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/** @noinspection PhpUnusedParameterInspection */
+declare(strict_types=1);
+
+namespace Flowpack\Media\Ui\GraphQL\Resolver\Type;
+
+/*
+ * This file is part of the Flowpack.Media.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Persistence\Doctrine\PersistenceManager;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\Media\Domain\Model\Adjustment\CropImageAdjustment;
+use Neos\Media\Domain\Model\ImageVariant;
+
+use t3n\GraphQL\ResolverInterface;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class AssetVariantResolver implements ResolverInterface
+{
+
+    /**
+     * @Flow\Inject
+     * @var PersistenceManager
+     */
+    public $persistenceManager;
+
+    /**
+     * @Flow\Inject
+     * @var ResourceManager
+     */
+    protected $resourceManager;
+
+
+    /**
```

Comment:
You can remove all the duplicate docblocks that don't add anything to the already defined types.